app/v2/notifications/post_notifications.py

```diff
@@ -144,6 +167,11 @@ def post_bulk():
 
     max_rows = current_app.config["CSV_MAX_ROWS"]
     check_sms_limit = current_app.config["FF_SPIKE_SMS_DAILY_LIMIT"]
+    epoch_seeding_bounce = current_app.config["FF_BOUNCE_RATE_SEED_EPOCH_MS"]
+    bounce_rate_v1 = current_app.config["FF_BOUNCE_RATE_V1"]
+    if bounce_rate_v1 and epoch_seeding_bounce:
+        _seed_bounce_data(epoch_seeding_bounce, str(authenticated_service.id))
```

Comment:
This is just going to seed the bounce rate in the event that a service tries to send. What about if the service does not send but is looking at their dashboard? Then they wouldn't see a valid bounce rate.
Would it make sense to move the "seeding" logic to a celery task that will get run for all services regardless of if they are sending?

Reply:
This is actually not such a big problem since we will only use this in the event that we lose data from Redis - definitely an edge case.